modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase

```diff
@@ -15,6 +15,59 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-9800. User can add a sample ULR in the virtual product's after the select file is removed"
```

Comment:
"This is a test for COMMERCE-9800. Users can add a sample URL to the virtual products after the selected file is removed."